.github/workflows/inference-test-unconditional.yml

```diff
@@ -0,0 +1,48 @@
+# Tests hitting the inference API are slow and prone to hanging, resulting in
+# timeouts. Therefore, we only run a subset of settings with inference tests by
+# default. See #118
+
+name: inference-tests
+
+on:
+  - push
+  - pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
we can also disable this when `[CI inference]` is in the commit message, since the other job does this as well.